components/notability/wikis/rainbowsix/notability_checker_config.lua

```diff
@@ -0,0 +1,207 @@
+		tier = 1,
+		options = {
+			dateLossIgnored = true,
+		},
+		tiertype = {
+			{
+				name = Config.TIER_TYPE_GENERAL,
+				points = 20000,
+			},
+			{
+				name = Config.TIER_TYPE_QUALIFIER,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_SHOW_MATCH,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_MONTHLY,
+				points = 50,
+			},
+			{
+				name = Config.TIER_TYPE_WEEKLY,
+				points = 25,
+			},
+		},
+	},
+	{
+		tier = 2,
+		options = {
+			dateLossIgnored = false,
+		},
+		tiertype = {
+			{
+				name = Config.TIER_TYPE_GENERAL,
+				points = 2000,
+			},
+			{
+				name = Config.TIER_TYPE_QUALIFIER,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_SHOW_MATCH,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_MONTHLY,
+				points = 50,
+			},
+			{
+				name = Config.TIER_TYPE_WEEKLY,
+				points = 25,
+			},
+		},
+	},
+	{
+		tier = 3,
+		options = {
+			dateLossIgnored = false,
+		},
+		tiertype = {
+			{
+				name = Config.TIER_TYPE_GENERAL,
+				points = 600,
+			},
+			{
+				name = Config.TIER_TYPE_QUALIFIER,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_SHOW_MATCH,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_MONTHLY,
+				points = 50,
+			},
+			{
+				name = Config.TIER_TYPE_WEEKLY,
+				points = 25,
+			},
+		},
+	},
+	{
+		tier = 4,
+		options = {
+			dateLossIgnored = false,
+		},
+		tiertype = {
+			{
+				name = Config.TIER_TYPE_GENERAL,
+				points = 350,
+			},
+			{
+				name = Config.TIER_TYPE_QUALIFIER,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_SHOW_MATCH,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_MONTHLY,
+				points = 50,
+			},
+			{
+				name = Config.TIER_TYPE_WEEKLY,
+				points = 25,
+			},
+		},
+	},
+	{
+		tier = 5,
+		options = {
+			dateLossIgnored = false,
+		},
+		tiertype = {
+			{
+				name = Config.TIER_TYPE_GENERAL,
+				points = 150,
+			},
+			{
+				name = Config.TIER_TYPE_QUALIFIER,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_SHOW_MATCH,
+				points = 0,
+			},
+			{
+				name = Config.TIER_TYPE_MONTHLY,
+				points = 50,
+			},
+			{
+				name = Config.TIER_TYPE_WEEKLY,
+				points = 25,
+			},
+		},
+	},
+}
+
+-- This function adjusts the score for the placement, e.g.
+-- a first placement should score more than a 10th placement.
+-- See also the EXTRA_DROP_OFF_TYPES.
+function Config.placementDropOffFunction(tier, tierType)
+	-- R6 is currently setting 0 points for the EXTRA_DROP_OFF types
+	-- but have plans to add points for them once modnotability is added on the wiki
+	if tierType ~= nil and Table.includes(Config.EXTRA_DROP_OFF_TYPES, tierType:lower()) then
+		return function(score, placement) return score / (placement * placement) end
+	end
+
+	return function(score, placement) return score / placement end
+end
+
+-- Adjusts the score to compensate for the mode, you might
+-- want to decrease the points given for a certain mode
+function Config.adjustScoreForMode(score, mode)
```

Comment:
this seems pretty pointless
it just returns the score
no need to define modeMod